### ISSUE TITLE
2passtools merge bug report

### ISSUE THREAD

**Issue by btrspg** [bartongroup/2passtools#5]
Hi there,

When I use 2passtools merge junction bedfiles from different samples, it raises an error like this:

```
  File "~/lib/python3.8/site-packages/lib2pass/merge.py", line 29, in <listcomp>
    res = [read_junc_bed(fn) for fn in junc_bed_fns]
  File ~/lib/python3.8/site-packages/lib2pass/merge.py", line 18, in read_junc_bed
    term = int(term)
UnboundLocalError: local variable 'term' referenced before assignment
```

I noticed there may be an error in here https://github.com/bartongroup/2passtools/blob/master/lib2pass/merge.py#L18. Hope you may fix it.

Thanks

**Comment by btrspg** [comment]
I have fixed the issue here in this PR #7

**Comment by mparker2** [comment]
Thanks @btrspg!
